autoload/indent_guides.vim

```diff
@@ -100,7 +100,7 @@ endfunction
 "
 function! indent_guides#highlight_colors() abort
   if s:auto_colors
-    if has('gui_running') || has('nvim')
+    if has('gui_running') || has('nvim') || has('termguicolors')
```

Comment:
This should in fact be:
```vim
    if has('gui_running') || has('nvim') || (has('termguicolors') && &termguicolors)
```
otherwise it doesn't work when `termguicolors` are disabled, as `indent_guides#gui_highlight_colors` sets `guifg`/`guibg` only.

Reply:
Makes sense! Would you mind sending that as a PR?

Reply:
Sure, https://github.com/preservim/vim-indent-guides/pull/155